pymemcache/client/base.py

```diff
@@ -864,7 +872,7 @@ def __init__(self,
 
     def check_key(self, key):
         """Checks key and add key_prefix."""
-        return _check_key(key, key_prefix=self.key_prefix)
+        return _check_key(key, allow_unicode_keys=self.allow_unicode_keys, key_prefix=self.key_prefix)
```

Comment:
I realized that we don't have an explicit style guidelines in the project. I will add a ticket to add one.
My personal preference is 120 lines, and this fits within that limit, so I am OK with it.